components/org.wso2.carbon.status.dashboard.web/src/distributedView/ParentAppTable.jsx

```diff
@@ -0,0 +1,177 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+import React from "react";
+
+//Material UI
+import {Table, TableBody, TableHeader, TableHeaderColumn, TableRow, TableRowColumn} from "material-ui/Table";
+import Pagination from "material-ui-pagination";
+import Circle from "material-ui/svg-icons/av/fiber-manual-record";
+import {TableFooter} from "material-ui/Table/index";
+import StatusDashboardAPIS from "../utils/apis/StatusDashboardAPIs";
+
+const dataConstants = {PAGE_LENGTH: 5};
+
+let currentPage = 1;
+
+/**
+ * class which manages Siddhi App list of a worker.
```

Comment:
Class level comment should be a sentence. Hence first letter (c) has be capital.